### PR TITLE
Add code from previous Microverse Trials for review

### PR DESCRIPTION
**Pull request by KingKowa**
The purpose of this project is to review code in a project from Microverse trials.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,7 @@
-# Trial-Review
\ No newline at end of file
+# Fred-s-Personal-Youtube-Page
+INTRODUCTION
+This project is about building a Youtube Page
+SKILL TO BE DEMONSTATED
+How to embed Youtube video player into a page
+Showing thumbnail images on the right side of the page
+Styling the page with CSS
```
Comment (KingKowa) [inline README.md:7]: README file is very descriptive. However, I think you can add other sections to make it complete. Follow the Microverse README template to complete the other [sections.](https://github.com/microverse/readme-template)

File: index.html [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    </nav>
+    
+    
+    
+    <iframe class="video" width="560" height="315" src="https://www.youtube.com/embed/Yg6UhhV_K1s" 
+            title="YouTube video player" 
+            frameborder="0" 
+            allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" 
+            allowfullscreen controls>
+    </iframe>
```
Comment (KingKowa) [inline index.html:23]: Kindly add the required semantic tag to this section.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <nav class="nav">
+        <a href="">video</a>
+        <a href="">sidebar</a>
+    </nav>
+
```
Comment (KingKowa) [inline index.html:15]: Kindly add the required semantic tag to this section.

**Review by KingKowa — COMMENTED** [review]
Changes Required.
Hi @KingKowa 

Great job so far. You are almost there. 
Kindly implement the required changes and
make sure there are no linter error.

Happy coding.

File: index.html (hunk 2 above)
Comment (youmari) [reply index.html:23]: @KingKowa Please kindly you should write  why you are required changes, and explain in detail why he should use semantic tag

File: index.html (hunk 3 above)
Comment (youmari) [reply index.html:15]: @KingKowa  same goes here for more detail check the comment above

File: index.html (hunk 2 above)
Comment (youmari) [inline index.html:23]: @KingKowa Please kindly you should write why you are required changes, and explain in detail why he should use semantic tag

File: index.html (hunk 3 above)
Comment (youmari) [inline index.html:15]: @KingKowa same goes here for more detail check the comment above

**Review by youmari — CHANGES_REQUESTED** [review]
Hi @KingKowa , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <nav class="nav">
+        <a href="">video</a>
+        <a href="">sidebar</a>
+    </nav>
```
Comment (KingKowa) [inline index.html:14]: Kindly use the required semantic tag for this section as this helps better define the different sections of your page and improves your page's section engine optimization

File: index.html (hunk 2 above)
Comment (KingKowa) [inline index.html:23]: Kindly use the required semantic tag for this section as this helps better define the different sections of your page and improves your page's section engine optimization

**Review by KingKowa — COMMENTED** [review]
Required Changes

Hi @KingKowa,

Good job so far. You have done extremely well. There are a few changes that you need
to do to finish this project. I have outlined those suggestions in the comments below. Kindly
ensure there are no linter errors before submitting for review.

Cheers and Happy coding.

File: index.html (hunk 4 above)
Comment (zieeco) [reply index.html:14]: - [ ] **Hi, you're doing great with the review but I want to get to know why you said you should use the semantic tag as the `<nav>` tag is a semantic tag. Could you please explain more in detail, which semantic tag you should use as it is very confusing to understand your review in this context which is contrary to the highlighted line of code?**

File: index.html (hunk 2 above)
Comment (zieeco) [reply index.html:23]: - [ ] **Hi, you're doing great with the review but I want to get to know why you said you should use the semantic tag as the `<nav>` tag is a semantic tag. Could you please explain more in detail, which semantic tag you should use as it is very confusing to understand your review in this context which is contrary to the highlighted line of code?**

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @KingKowa 


Great work on making the changes requested by the previous reviewer 👏 
You've done well implementing some of the changes, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- Impressive beginning.✔️


## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 4 above)
Comment (KingKowa) [inline index.html:14]: Kindly wrap the <nav> tag in a <header> tag as this helps better define the different sections of your page and improves your page's search engine optimization.

File: index.html (hunk 2 above)
Comment (KingKowa) [inline index.html:23]: Kindly wrap the <iframe> tag in the <main> tag as this helps define the different sections of your page and improves your page's search engine optimization.

**Review by KingKowa — COMMENTED** [review]
Changes Required.
Hi @KingKowa

Great job so far. You are almost there.
Kindly implement the required changes and
make sure there are no linter error.

Happy coding.

File: index.html (hunk 4 above)
Comment (Maclenn77) [reply index.html:14]: - [ ] Great job mentioning this to your student in a polite way and adding an explanation. However, the comment is still unclear. Which kind of tag should the student use? A `<header>`  tag? A `<div>` tag? Or a `<main>` tag? Giving more context and information can help the student to address the problem.

**SEE AN EXAMPLE BELOW**:

_I kindly suggest that to wrap the `<nav>` tag in a `<header>` tag  as this helps better define the different sections of your page and improves your page's search engine optimization._

[OPTIONAL]  There's an unnecessary break of line in the comment. Kindly address this issue so your comment can look more professional.

File: index.html (hunk 2 above)
Comment (Maclenn77) [reply index.html:23]: - [ ] Great job mentioning this to your student in a polite way and adding an explanation. However, the comment is still unclear as the comment above. Which kind of tag should the student use? Giving more context and information can help the student to address the problem.

**SEE AN EXAMPLE BELOW**:

_I kindly suggest that to wrap the `<nav>` tag in a `<whatever>` tag as this helps better define the different sections of your page and improves your page's search engine optimization._

[OPTIONAL] There's an unnecessary break of line in the comment. Kindly address this issue so your comment can look more professional.

**Review by Maclenn77 — CHANGES_REQUESTED** [review]
Hi @KingKowa, 

Great work on making the changes requested by a previous reviewer 👏 .

You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

Almost done

 ## Highlights

- You use Github inline comments feature correctly. :+1:
- You use polite suggestions instead of orders. :bowtie:

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

[OPTIONAL] You could use markdown syntax to **highlight** something or distinguish a `code line` from text. The basic Markdown syntax is available [here](https://www.markdownguide.org/basic-syntax/).

[OPTIONAL] You could include emojis and gifs to make your code review more human-friendly :alien: .


Gif example



_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 4 above)
Comment (KingKowa) [inline index.html:14]: I kindly suggest you wrap the `nav` tag in a `header` tag as this helps better define the different sections of your page and improves your page's search engine optimization

File: index.html (hunk 2 above)
Comment (KingKowa) [inline index.html:23]: I kindly suggest you wrap the `iframe` tag in a `main` tag as this helps better define the different sections of your page and improves your page's search engine optimization

**Review by KingKowa — COMMENTED** [review]
Changes Required.
Hi @KingKowa

Great job so far. You are almost there.
Kindly implement the required changes and kindly
check for linter error.

Happy coding.

**Review by OLIPLICHE — APPROVED** [review]
Hi @KingKowa , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**